Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -95,7 +103,7 @@ lemma iff_rTensor_injective :
     erw [← Equiv.comp_injective _ (TensorProduct.lid R M).toEquiv]
     have h₁ := F.out hI
     rw [← aux] at h₁
-    exact h₁
+    refine h₁
```

Comment:
Is this wanted?

Reply:
by accident, I think i messed the merging process somehow.